R/module_teal_with_splash.R

```diff
@@ -21,11 +21,18 @@
 #'
 #' @param id (`character(1)`)
 #'   module id
+#' @param lockfile_task [`future::future`] promise invoked with [`create_renv_lockfile`]
 #' @inheritParams init
 #' @param modules (`teal_modules`) object containing the output modules which
 #'   will be displayed in the `teal` application. See [modules()] and [module()] for
 #'   more details.
 #' @inheritParams shiny::moduleServer
+#'
+#' @section Reproducibility:
+#' Reproducibility is supported by multiple features. `teal` includes a `utils::sessioInfo()` output to allow to compare
+#' packages used in the session. It also allows to create `renv` `.lockfile` to support project setup reproducibility.
+#' For more information about `.lockfile` creation visit [`create_renv_lockfile()`].
```

Comment:
is it ok to introduce ref to a private function?

Reply:
CRAN policy does not disallow this. We need to get some documentation on this process somewhere, so that users know how to steer it :)